src/App.jsx

```diff
@@ -9,7 +10,32 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+
+  <Article
+    title="Electric-propulsion startup Orbion raises $9.2 million Series A"
```

Comment:
```suggestion
    title={article.title}
```
Why did you copy text? What if text in my article object will be modified?
Change it for other props by yourself

Reply:
Thanks :)